app/controllers/task_controller.rb

```diff
@@ -51,4 +80,11 @@ def sort_completed
 
     render :home
   end
+
+  def sort_name
```

Comment:
I think this is duplicate code from line 70 above. In any event, it replaces the method of the same name on line 70, so you should only keep one of the two definitions.